Configuration/TypoScript/setup.txt

```diff
@@ -1,8 +1,8 @@
 plugin.tx_servicebw2_servicebw {
   view {
-    templateRootPath = {$plugin.tx_servicebw2_servicebw.view.templateRootPath}
-    partialRootPath = {$plugin.tx_servicebw2_servicebw.view.partialRootPath}
-    layoutRootPath = {$plugin.tx_servicebw2_servicebw.view.layoutRootPath}
+    templateRootPaths.10 = {$plugin.tx_servicebw2_servicebw.view.templateRootPath}
+    partialRootPaths.10 = {$plugin.tx_servicebw2_servicebw.view.partialRootPath}
+    layoutRootPaths.10 = {$plugin.tx_servicebw2_servicebw.view.layoutRootPath}
```

Comment:
What do you thing about such a solution?
https://github.com/georgringer/news/blob/master/Configuration/TypoScript/setup.txt#L8-L21